samples/net/echo_server/boards/qemu_cortex_m3.conf

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Same here, TAP connectivity no longer posible with qemu_cortex_m3

Reply:
Retaining the original sample, and added separate overlay.